### ISSUE TITLE
How can I compare the results with selfRAG?

### ISSUE THREAD

**Issue by hummingbird2030** [starsuzi/Adaptive-RAG#6]
Thanks for your work! I wonder how can I run selfRAG on your datasets?
How to calculate the confusion metric in Fig.3?

**Comment by starsuzi** [comment]
Hello,
Thanks for your interest!

We ran selfRAG by following the implementation from the original GitHub repository, using the publicly open checkpoints.
Also, we calculated the confusion matrix with `from sklearn.metrics import confusion_matrix`.
